core/postfix/start.py

```diff
@@ -66,6 +66,12 @@ def is_valid_postconf_line(line):
     os.system("postmap {}".format(destination))
     os.remove(destination)
 
+if not os.path.exists("/etc/postfix/tls_policy.map.db"):
```

Comment:
When testing I see that postfix creates the file tls_policy.map.lmdb instead of tls_policy.map.db.